charts/istio-operator/CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 ### Removed -->
 
+## v2.1.3 - 2021-03-08
+
+## Changed
+
+- Upgraded _Istio_ version to _v1.9.1_
+
```

Comment:
```suggestion
### Changed

- Upgraded _Istio_ version to _v1.9.1_

## v2.1.2 - 2021-02-11

### Changed

- Update docs

```

Could you make this change to fix a minor markdown typo and to also add a changelog entry I forgot to add?